clang/lib/Driver/Driver.cpp

```diff
@@ -2408,11 +2410,14 @@ void Driver::PrintHelp(bool ShowHidden) const {
 llvm::Triple Driver::MakeSYCLDeviceTriple(StringRef TargetArch) const {
   SmallVector<StringRef, 5> SYCLAlias = {
       "spir",       "spir64",  "spir64_fpga", "spir64_x86_64",
-      "spir64_gen", "spirv32", "spirv64"};
+      "spir64_gen", "spirv32", "spirv64",     "nvptx64"};
```

Comment:
Looking at some of the test cleanup - is there any reason why `nvptx` cannot be supported?

Reply:
Since `nvptx64-nvidia-cuda` is the only officially documented/supported value for NVidia GPU targets, we are adding support only for '`nvptx64`'